docs/conf.py

```diff
@@ -66,7 +66,7 @@
 
 # General information about the project.
 project = u"google-cloud-automl"
-copyright = u"2017, Google"
+copyright = u"2019, Google"
```

Comment:
Not sure if these should be part of this commit - this was what I was talking about with the copyright change

Reply:
👍 Good call. Copyright years need to be addressed across the board (in source files and in docs). Client libraries folks are waiting to hear back on how to address in the libraries.